ext/node/polyfills/_brotli.js

```diff
@@ -28,7 +34,9 @@ const toU8 = (input) => {
     return enc.encode(input);
   }
 
+  // deno-lint-ignore prefer-primordials
   if (input.buffer) {
+    // deno-lint-ignore prefer-primordials
     return new Uint8Array(input.buffer);
   }
```

Comment:
Note: using `TypedArrayPrototypeGetBuffer` here causes `node_unit_tests::zlib_test` to fail. Hence the ignores.

Reply:
@petamoriken, does this look right to you?

Reply:
Maybe it is necessary to detect TypedArray or DataView.

https://github.com/denoland/deno/blob/8dd7beb7c6b775d0952c6be0e458066d4cb7ad09/ext/web/08_text_encoding.js#L125-L131

Reply:
That makes sense and worked! Thank you.